docs/ruleset_functions/zone_meets_G3_1_1e.md

```diff
@@ -0,0 +1,60 @@
+# zone_meets_G3_1_1e
+**Schema Version:** 0.0.22  
+
+**Description:** determines whether a given zone meets the G3_1_1e exception "Thermal zones designed with heating-only systems in the proposed design serving storage rooms, stairwells, vestibules, electrical/mechanical rooms, and restrooms not exhausting or transferring air from mechanically cooled thermal zones in the proposed design shall use system type 9 or 10 in the baseline building design."
+
+**Inputs:**
+- **P-RMR**
+- **B-RMR**
+- **zone**
+
+**Returns:**  
+- **result**: a string - either "E" or "No"
+ 
+**Function Call:**
+- **is_a_vestibule**
+
+## Logic:
+- create list of eligible space types: `eligible_space_types = ["STORAGE_ROOM_HOSPITAL", "STORAGE_ROOM_SMALL", "STORAGE_ROOM_LARGE", "WAREHOUSE_STORAGE_AREA_MEDIUM_TO_BULKY_PALLETIZED_ITEMS", "WAREHOUSE_STORAGE_AREA_SMALLER_HAND_CARRIED_ITEMS", "STAIRWELL", "ELECTRICAL_MECHANICAL_ROOM", "RESTROOM_FACILITY_FOR_THE_VISUALLY_IMPAIRED", "RESTROOM_ALL_OTHERS"]
+- set the result variable to "No" - only a positive test can give it a different value: `result = "No"`
+- set eligibility boolean to False: `eligible = TRUE`
+- For each space in zone: `for space_p in zone.Spaces:`
+	- check if the space has an eligible lighting type: `if space_p.lighting_space_type not in eligible_space_types:`
+		- set eligibility to False - any non-compliant space type will result in a non-eligible zone: `eligible = FALSE`
+
+- if the zone is not eligible, check if it is a vestibule: `if eligible == FALSE:`
+	- set variable is_a_vestibule = is_zone_a_vestibule: `is_a_vestibule = is_zone_a_vestibule(zone, B-RMR)`

```

Comment:
Does this function has to take in `B-RMD`?
If geometry is the same, I think `P-RMD` could work? By setting it to `P-RMD`, we can save one argument.

Reply:
good point.  changed.